src/pages/ReportDetailsPage.tsx

```diff
@@ -332,7 +357,7 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
                             />
                         </OfflineWithFeedback>
                     )}
-                    {isGroupChat && <ChatDetailsQuickActionsBar report={report} />}
+                    {!isGroupDMChat && <ChatDetailsQuickActionsBar report={report} />}
```

Comment:
We can pin a Group DM chat, but not share it. They are getting deprecated soon. But, I think this is more or less incorrect logic and we should be able to remove the `!isGroupDMChat`.